test/user.test.js

```diff
@@ -41,12 +41,12 @@ describe('User model', function () {
       user.email = null;
 
       return user.validate()
-      .then(function () {
+        .then( function ( ) {
         throw new Error('validation should fail without email');
-      }, function (err) {
-        expect(err).to.be.an.instanceOf(Error);
-        expect(err.message).to.contain('email cannot be null');
-      });
+        }, function ( err ) {
+          expect( err ).to.be.an.instanceOf( Error );
+          expect(err.errors[0].type).to.contain('notNull Violation');
```

Comment:
alternatively:
`expect(err.message).to.contain('Validation Error');`
seems to work too